sdk/python/foundation-models/system/inference/text-to-image/scoring-files/score/score_batch.py

```diff
@@ -0,0 +1,584 @@
+        raise Exception("Error reading model signature: " + ml_model.__str__() + str(e))
+
+
+def get_output_schema(model_path: str) -> Schema:
+    """Get output schema from model signature.
+
+    :param model_path: path to model
+    :type model_path: str
+    :raises Exception: Error reading model signature
+    :return: output schema
+    :rtype: MLflow.types.Schema
+    """
+    try:
+        with open(os.path.join(model_path, "MLmodel"), "r") as stream:
+            ml_model = yaml.load(stream, Loader=yaml.BaseLoader)
+        if "signature" in ml_model:
+            if "outputs" in ml_model["signature"]:
+                return Schema.from_json(ml_model["signature"]["outputs"])
+        g_logger.warn(
+            "No signature present in MLmodel file: "
+            + ml_model.__str__()
+            + "\nSignatures are encouraged, but AzureML will take a "
+            + "best effort approach to loading the data files."
+        )
+    except Exception as e:
+        raise Exception("Error reading model signature: " + ml_model.__str__() + str(e))
+
+
+def get_dtypes(schema):
+    try:
+        if schema is None:
+            return None
+        elif schema.is_tensor_spec():
+            data_type = schema.numpy_types()[0]
+            data_shape = schema.inputs[0].shape
+            g_logger.info(
+                f"Enforcing tensor type : {data_type} and shape: {data_shape}"
+            )
+            return data_type, data_shape
+        else:
+            column_dtypes = dict(zip(schema.input_names(), schema.pandas_types()))
+            g_logger.info("Enforcing datatypes:" + str(column_dtypes))
+            return column_dtypes
+    except Exception as e:
+        raise Exception(
+            "Error reading types from model signature schema: "
+            + schema.__str__()
+            + str(e)
+        )
+
+
+def _get_data_loader(file_extension):
+    loader = g_file_loader_dictionary.get(file_extension)
+    if loader is None:
+        raise ValueError(
+            "File type '" + str(file_extension) + "' is not supported. "
+            f"Current accepted filetypes are: {list(g_file_loader_dictionary.keys())}"
+        )
+    else:
+        return loader
+
+
+def load_data(data_file):
+    _, file_extension = os.path.splitext(data_file)
+    data_loader = _get_data_loader(file_extension)
+    return data_loader(data_file)
+
+
+def load_csv(data_file):
+    if g_schema_input and g_schema_input.is_tensor_spec():
+        g_logger.info(
+            "Reading csv input file into numpy array because the model specifies tensor input signature."
+        )
+        return np.genfromtxt(data_file, dtype=g_dtypes)
+    else:
+        g_logger.info(
+            "Reading csv input file into pandas dataframe because the model specifies column based input signature or no signature."
+        )
+        return pd.read_csv(data_file, engine="c", dtype=g_dtypes)
+
+
+def _load_image_as_array(data_file):
+    """
```

Comment:
Let's ensure to use same docstring style across the board. We have two styles within this file.

Reply:
This docstring was already present in original code of batch deployment team.